api4/team.go

```diff
@@ -1275,12 +1275,23 @@ func inviteUsersToTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	emailList := model.ArrayFromJSON(r.Body)
+	memberInvite := &model.MemberInvite{}
+	if jsonErr := json.NewDecoder(r.Body).Decode(memberInvite); jsonErr != nil {
```

Comment:
Would this be considered a breaking change to the api?

Reply:
yes, because the api call was expecting before an array of strings and now expect a struct defining at least the emails property which is that array of strings. I think is a better idea to support both data structures so we don't break the api current behavior.